lib/progress_dialog.dart

```diff
@@ -216,6 +236,20 @@ class ProgressDialog {
                               ? Alignment.bottomRight
                               : Alignment.bottomCenter,
                         )
+                      : Container(),
+                  isCancelable
+                      ? InkWell(
+                          onTap: onCancelClicked,
+                          child: Padding(
+                            padding: const EdgeInsets.all(8.0),
+                            child: Text(
```

Comment:
This text seems to inflate the dialog, maybe we should just add an "X" button.

Reply:
From a UX perspective, I think the ``X`` button refers to closing the dialog and keeping everything working in the background.
You can use `cancelFontSize` to set the cancel text size.